src/components/va-accordion/va-accordion.tsx

```diff
@@ -68,6 +68,8 @@ export class VaAccordion {
           header: clickedItem.header,
           subheader: clickedItem.subheader,
           level: clickedItem.level,
+          sectionHeading: this.sectionHeading,
+          bordered: this.bordered
```

Comment:
Is including `bordered` here intentional?

Reply:
Weird.  It was not.  Removed!